pages/index.tsx

```diff
@@ -1,20 +1,57 @@
-import Head from 'next/head'
+import Head from 'next/head';
+import ClayNav from '@clayui/nav';
+import ClayIcon from '@clayui/icon';
+
+const spritemap = '../images/icons.svg';
 
 export default function Home() {
-  return (
-    <div>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <main>
-       
-      </main>
-
-      <footer>
-      </footer>
-    </div>
-  )
+	return (
+		<>
+			<Head>
+				<title>LutBot</title>
+
+				<meta
+					name="description"
+					content="Generated by create next app"
+				/>
+				<link rel="icon" href="/favicon.ico" />
+			</Head>
+
+			<header>
+				<ClayNav>
+					<ClayNav.Item>
+						<ClayNav.Link active href="/">
+							<ClayIcon symbol="sheets" spritemap={spritemap} />
```

Comment:
Instead of using the spritemap on each item, let's wrap the _app file with ClayContextProvider and that way icons will automatically inherit the spritesheet